src/Views/FirmwareView.vala

```diff
@@ -216,7 +215,7 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
     private void header_rows (Widgets.FirmwareUpdateRow row1, Widgets.FirmwareUpdateRow? row2) {
         if (row2 == null && row1.is_updatable) {
             var header = new FirmwareHeaderRow (
-                dngettext (GETTEXT_PACKAGE, "%u Update Available", "%u Updates Available", num_updates).printf (num_updates)
+                dngettext (GETTEXT_PACKAGE, "%u available firmware upgrade", "%u available firmware upgrades", num_updates).printf (num_updates)
```

Comment:
```suggestion
                dngettext (GETTEXT_PACKAGE, "One available firmware upgrade", "%u available firmware upgrades", num_updates).printf (num_updates)
```
Not sure if this causes issues for any translators.

Reply:
This isn't the correct way to use ngettext. It needs to stick with the `%u` placeholder

Reply:
Good to know. Will it then display as "1" or "One"?
I suppose a way to avoid it is to use a developer locale and then translate it in en_UK and en_US and so on as "One"?
I have been using "Én" in nb_NO since that is the more correct way to do it.

Reply:
In US English it will display as "1", which is intended for that locale

Reply:
Maybe the jury is out on that https://www.grammarly.com/blog/when-to-spell-out-numbers/ (this seems to be my position).
In the very least you would need a list of genders to pick the right one (intended ;) word to spell it out with in other languages.